### ISSUE TITLE
Работает по WiFi но не работает по проводу

### ISSUE THREAD

**Issue by Esembe1** [Flowseal/zapret-discord-youtube#764]
Здравствуйте, подскажите пожалуйста, почему все работает при подключении к WiFi, но не работает при подключении к тому же роутеру по проводу...

**Comment by Eptiy** [comment]
Попробуй перезапустить роутер и пк одновременно.

**Comment by johnnyd-78** [comment]
Попробуйте в батнике убрать =2 после --dpi-desync-autottl
Вряд ли использование дельты 2 - это хорошее решение

**Comment by Eptiy** [comment]
https://github.com/Flowseal/zapret-discord-youtube/discussions/252#discussioncomment-11251795

**Comment by Esembe1** [comment]
> Попробуй перезапустить роутер и пк одновременно.

Не помогло

**Comment by Esembe1** [comment]
> Попробуйте в батнике убрать =2 после --dpi-desync-autottl Вряд ли использование дельты 2 - это хорошее решение

Не помогло

**Comment by Eptiy** [comment]
Попробуй выполнить действия, которые я посоветовал в коменте
https://github.com/Flowseal/zapret-discord-youtube/discussions/252#discussioncomment-11252213

Вкратце: нужно привязать пк по маку один локальный айпи, этот айпи прописать в dmz.

**Comment by Eptiy** [comment]
Вообще, после того, как я в том обсуждении двум людям помог, можно заявлять, что дело тут не в запрете или системе, а в роутере. Увы, какие конкретно махинации с ним нужно произвести - выяснить не удалось. Я в данном случае отталкиваюсь от настроек на своём роутере. Кроме привязки айпи и dmz у меня ничего не настроено.

**Comment by Esembe1** [comment]
> Попробуй выполнить действия, которые я посоветовал в коменте [#252 (reply in thread)](https://github.com/Flowseal/zapret-discord-youtube/discussions/252#discussioncomment-11252213)
> 
> Вкратце: нужно привязать пк по маку один локальный айпи, этот айпи прописать в dmz.

![image](https://github.com/user-attachments/assets/a61bcd2e-99d9-4a67-b463-10e6c8621d88)
![image](https://github.com/user-attachments/assets/5e1ebc81-7fc3-4731-ac43-a6372eea956c)
![image](https://github.com/user-attachments/assets/4e5bd3e3-29b3-4649-9d08-7765c41abfb0)
Правила добавил еще и в домашнюю и в Ethernet. 
это то? или что то еще нужно сделать? просто не нашел именно пункта DMZ у себя в роутере...

**Comment by Eptiy** [comment]
Не совсем. Ты добавил правила в фаервол. Это может помочь, но я не уверен, потому что есть ещё NAT.

В общем, проверяй хотя бы так.

**Comment by Esembe1** [comment]
> Не совсем. Ты добавил правила в фаервол. Это может помочь, но я не уверен, потому что есть ещё NAT.
> 
> В общем, проверяй хотя бы так.

ну так уже проверил, не помогло(

**Comment by Eptiy** [comment]
Это печально.

Роутер подключён напрямую к кабелю провайдера, или через другое устройство? 

Например, у ростелеком сейчас почти везде оптоволокно, его нельзя к своему роутеру подключить технически, поэтому многие подключают свой через провайдеровский.

**Comment by Eptiy** [comment]
Кстати, проверь, что пк взял установленный ip.  Мне этот интерфейс вообще не знаком, и я не знаю, как твой роутер работает, но у меня при привязке ip в DHCP фактически ip устройства меняется только после истечения аренды, либо перезагрузки роутера и устройства одновременно.

**Comment by Esembe1** [comment]
> Это печально.
> 
> Роутер подключён напрямую к кабелю провайдера, или через другое устройство?
> 
> Например, у ростелеком сейчас почти везде оптоволокно, его нельзя к своему роутеру подключить технически, поэтому многие подключают свой через провайдеровский.

Да, кабель подключен напрямую, ip я пк прописал постоянный тот, который у него и был, и проверил, ip не поменялся.
добавил еще вот такую штуку, в официальном руководстве kenetik написано что как раз тут и не действует NAT
![image](https://github.com/user-attachments/assets/b1320a9f-93ef-43d8-8aa3-de468a3e228f)
![image](https://github.com/user-attachments/assets/b6357c3c-caa5-4c0f-b123-c0d0b6d4bb7f)
и вообще ничего не помогает уже...

**Comment by Eptiy** [comment]
Сейчас возник вопрос: как там привязывается ip? У меня он вяжется по маку, соответственно он привязан к сетевой карте, и если я всё перечисленное настрою, прописав мак wi-fi адаптера, то по проводу оно работать не будет.

**Comment by Esembe1** [comment]
> Сейчас возник вопрос: как там привязывается ip? У меня он вяжется по маку, соответственно он привязан к сетевой карте, и если я всё перечисленное настрою, прописав мак wi-fi адаптера, то по проводу оно работать не будет.

ну тут идет привязка к устройству как я понимаю по маку. я уже думаю, может все таки сама сетевуха что то блочит у меня

**Comment by Eptiy** [comment]
> ну тут идет привязка к устройству как я понимаю по маку.

Т.е. ты прописывал мак проводной сетевухи? Если так, то все правила должны на неё распространяться.

>  уже думаю, может все таки сама сетевуха что то блочит у меня.

Тоже не исключено, но с такими случаями не сталкивался, помочь не могу.

**Comment by Esembe1** [comment]
> Если так, то все правила должны

в списке устройств уже висел мой пк с маком его, я просто задал ему перманентный ip, но мак я проверил, точно от самого пк. ладно, спасибо за помощь)
